### ISSUE TITLE
Implement CLASS selection

### ISSUE THREAD

**Issue by frons79** [Futil/foundry-mothership#25]
During character creation, CLASS is an open text-type field, instead it should accept only a close selection of values (i.e. the core classes).

**Comment by hollowphoton** [comment]
Will look into this request soon

**Comment by Futil** [comment]
I don't think that restricting input is particularly useful for most player groups. I like to ensure that people have the tools to easily homebrew or change values without getting into the weeds of adjusting values off-sheet.

Having a class item type that you can drag onto the sheet that autofills the type could be useful, but I'm not sure closing down the text field itself is good from my experience in other systems that do that.

I am curious to hear your response.

**Comment by frons79** [comment]
Well, drag'n'drop a class item is fine if you wish to leave freedom to type
in whatever details one wants.
I think it's a good compromise between freedom and getting the advantages
of automation.

𝗚𝗮𝗯𝗿𝗶𝗲𝗹𝗲 𝗙𝗿𝗼𝗻𝘁𝗶𝗻𝗶
e-mail: ***@***.***
discord: frons79

Il gio 9 nov 2023, 16:47 Futil ***@***.***> ha scritto:

**Comment by Raggitator** [comment]
I would, humbly, second this. If we could at least get a check box for Androids one could make Android blood white as per Alien Standard in Splatter-mod. It would be kick-ass. <3

**Comment by Melon-PieldeSapo** [comment]
It will feature a drop-down selector of predefined class that apply the apropiate modifier but also it will be an open text field.
(drag-and-drop) feature will be done aswell.

**Comment by Melon-PieldeSapo** [comment]
Android checkbox is also in the works with the new class system.

**Comment by hollowphoton** [comment]
This is now available, either with the PSG module installed or other modules with community created classes.
